tank_battalion/game_state.pde

```diff
@@ -168,7 +174,7 @@ class GameState extends State
     
     for(int i = 0; i < n_lives; i++)
     {
-      image(tank_image, width - 300 + i * 100, 300, 64, 64);  
+      image(tank_image, width - 340 + i * 64, 600, 64, 64);  
```

Comment:
Unrelated to this issue, but it's fine.

Reply:
That was actually from his previous issue which he forgot to commit. :wink: (I helped him)